### ISSUE TITLE
[Bug]: Client Crash when viewing AE2 Fluid Tank crafting recipe

### ISSUE THREAD

**Issue by ChiefArug** [Laskyyy/Create-Astral#422]
### Bug Description

I clicked on the fluid tank in REI and it crashed. Nuff' said. 

### Version

2.1.2

### Logs or additional context

https://gist.github.com/ChiefArug/b1d812cf001a11b6ad7271b10ddb0a45

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the modpack
- [X] I have verified that I have followed the required steps to set up the modpack

**Comment by The-Shortman** [comment]
I do not believe such an item exists, or at least, it isn't viewable in REI, but all the fluid-related AE2 items that are visible do not cause crashes in 2.1.3

**Comment by ChiefArug** [comment]
I wasnt able to ever reproduce it, it was just a one off crash.
